deployment/Dockerfile

```diff
@@ -79,10 +79,8 @@ RUN apt-get update -y && \
 ####################
 # Download mathjax (same version as plotly.js extras/)
 
-RUN apt-get update -y && apt-get install -y subversion && \
-    rm -rf /var/lib/apt/lists/* && apt-get clean && \
-    svn checkout https://github.com/plotly/plotly.js.git/trunk/dist/extras/mathjax && \
-    apt-get --auto-remove -y remove subversion
+RUN curl -L https://github.com/plotly/plotly.js/archive/master.tar.gz \
+    | tar -xvzf - --strip-components=3 plotly.js-master/dist/extras/mathjax
 
```

Comment:
Replacing `svn` by `git` is a major win because the `svn checkout` from Github often failed!

Reply:
Awesome :tada: